bundle/src/main/java/com/adobe/acs/commons/httpcache/engine/CacheContent.java

```diff
@@ -132,8 +132,10 @@ public CacheContent() {
      * Construct from the custom servlet response wrapper..
      *
      * @param responseWrapper
+     * @deprecated
      * @return
      */
+    @Deprecated
     public CacheContent build(HttpCacheServletResponseWrapper responseWrapper) throws HttpCacheDataStreamException {
         this.status = responseWrapper.getStatus();
```

Comment:
Should just call non-deprecated `build(..)` method, rather than duplicating work.

```suggestion
          // Extracting HTTP Response Header Names and Values
         Map<String, List<String>> headers = new HashMap<>();
        List<String> headerNames = new ArrayList<>();

        headerNames.addAll(responseWrapper.getHeaderNames());
        for (String headerName: headerNames) {
            List<String> values = new ArrayList<>();
            values.addAll(responseWrapper.getHeaders(headerName));
            headers.put(headerName, values);
        }

        return build(responseWrapper, responseWrapper.getStatus(), responseWrapper.getCharacterEncoding(), responseWrapper. getContentType(), headerNames);
```